docs/README.md

```diff
@@ -1402,14 +1402,35 @@ export class UserReadModel {
   public constructor(readonly username: string, /* ...(other interesting fields from users)... */) {}
   
   @Projects(User, 'id')
-  public static projectUser(entity: User, current?: UserReadModel) { // Here we update the user fields}
+  public static projectUser(entity: User, current?: UserReadModel): ProjectionResult<UserReadModel> { // Here we update the user fields}
 
   @Projects(Post, 'ownerId')
-  public static projectUserPost(entity: Post, current?: UserReadModel) { //Here we can adapt the read model to show specific user information related with the Post entity}
+  public static projectUserPost(entity: Post, current?: UserReadModel): ProjectionResult<UserReadModel> { //Here we can adapt the read model to show specific user information related with the Post entity}
 }
 ```
 In the previous example we are projecting the `User` entity using the user `id` and also we are projecting the `User` entity based on the `ownerId` of the `Post` entity. Notice that both join keys are references to the `User` identifier, but it's not required that the join key is an identifier.
 
+As you may have notice from the `ProjectionResult` type, projections can also return `ReadModelAction`, which includes:
+
+1. Deletion of read models by returning the `ReadModelAction.Delete` value
+2. You can also use `ReadModelAction.Nothing` to keep the read model as it is too!
```

Comment:
Trying to avoid "also" + "too" in the same sentence and refrasing a bit:
```suggestion
2. You can also return `ReadModelAction.Nothing` to keep the read model untouched
```